app/src/at-trinhnguyen/java/asiantech/internship/winter/javacore/Exercise11.kt

```
@@ -0,0 +1,84 @@
+                while (j < n) {
+                    print(" " + array[j])
+                    j++
+                }
+                i = n - 2
+                while (i >= 0) {
+                    check = 1
+                    if (array[i] < array[i + 1]) {
+                        if (i == n - 2) {
+                            temp = array[i]
+                            array[i] = array[n - 1]
+                            array[n - 1] = temp
+                            break
+                        } else {
+                            //Tim so a[k] nho nhat ma >a[i] trong cac so ben phai a[i]
+                            k = i + 1
+                            j = i + 1
+                            while (j < n) {
+                                if (array[i + 1] > array[j] && array[j] > array[i]) k = j
+                                j++
+                            }
+                            //Doi cho a[k] va a[i]
+                            temp = array[i]
+                            array[i] = array[k]
+                            array[k] = temp
+                            //Sap xep lai tu a[i+1] toi a[n]
+                            j = i + 1
+                            while (j < n) {
+                                for (m in i + 1 until n) {
+                                    if (array[j] < array[m]) {
+                                        temp = array[j]
+                                        array[j] = array[m]
+                                        array[m] = temp
+                                    }
+                                }
+                                j++
+                            }
+                            break
+                        }
+                    } else {
+                        check = 0
+                        // break;
+                    }
+                    i--
+                }
+                //if(i==0)check=0;
+            }
+        } catch (e: Exception) {
```

Remove `try - catch` if empty catch block.